### DISCUSSION TITLE
Academy not accepting correct answers in Workflows

### DISCUSSION THREAD

**Discussion by sszymkowiczpg** [pimcore/pimcore#17735] [Q&A / Support] [answered]
I believe I have a correct answer to this question but still, Academy doesn't approve them. 
<img width="465" alt="image" src="https://github.com/user-attachments/assets/4c860475-168e-4cfa-9313-8640d84b2df2">

**Comment by fashxp** [comment, marked as answer]
symfony container needs to rebuilt after workflow configuration changes is correct too.

**Reply by sszymkowiczpg** [reply]
Could you please point a documentation reference?

**Reply by fashxp** [reply]
Configuration takes place in [symfony configuration](https://pimcore.com/docs/platform/Pimcore/Workflow_Management/Configuration_Details/) ... so it should be quite obvious that a container rebuild is necessary to apply changes. 

Additionally, there is a hint in [workflow designer documentation](https://pimcore.com/docs/platform/Worfklow_Designer/Workflow_Configuration/#:~:text=To%20apply%20workflow%20changes%20to%20Pimcore%2C%20the%20symfony%20container%20needs%20to%20be%20rebuilt.%20On%20development%20systems%20(in%20dev%20environment)%20this%20might%20happen%20automatically%20just%20when%20saving%20the%20workflow%20definition).
